runtime/service.go

```diff
@@ -183,9 +181,10 @@ func NewService(shimCtx context.Context, id string, remotePublisher shim.Publish
 		}
 	}
 
-	if cfg.Debug {
-		logrus.SetLevel(logrus.DebugLevel)
-		logger.Logger.SetLevel(logrus.DebugLevel)
+	logrusLevel, ok := cfg.DebugHelper.GetFirecrackerContainerdLogLevel()
+	if ok {
+		logrus.SetLevel(logrusLevel)
+		logger.Logger.SetLevel(logrusLevel)
 	}
```

Comment:
How about returning the default log level, rather than using `ok`? Unlike SDK, clients cannot pass a Logrus logger to the shim. So we don't have to honor the existing log level.

Reply:
Well, users may want to disable logging which that would prevent